recipes/spiceql/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "spiceql" %}
+{% set version = "1.0.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: .
+    url: https://code.usgs.gov/astrogeology/spiceql/-/archive/{{ version }}/spiceql-{{ version }}.tar.gz
+    sha256: df51ef41839b8d4e09153922ddf272c2da862fc0f126b35d978d8ff81ce8044e
+
+  # submodules
```

Comment:
Instead of using submodules, you should be using the respective conda packages.

Reply:
these are pretty lightweight header only libraries or not on conda yet. How necessary is it that they are all conda packages?

Reply:
Header only would be okish but otherwise they should be on conda-forge.

Reply:
This still needs to be addressed, either by include their licenses below or by using the conda packages.